packages/p2p/src/components/page-return/page-return.tsx

```diff
@@ -10,9 +10,9 @@ type TPageReturnProps = {
 
 const PageReturn = ({ className = '', onClick, page_title }: TPageReturnProps) => {
     return (
-        <div className={classNames('page-return', className)} data-testid='dt_page_return'>
-            <div onClick={onClick} className='page-return__button'>
-                <Icon icon='IcArrowLeftBold' size={16} />
+        <div className={classNames('page-return', className)}>
+            <div onClick={onClick} className='page-return__button' data-testid='dt_page_return_icon'>
```

Comment:
oh man this will affect my test case hahahai think you can still leave this as `dt_page_return` but below in Icon you can pass `data-testid='dt_page_return_icon'`

Reply:
yes... it actually conflicts with one of my existing refactoring pr as well..  i should have done this change in page-return refactor itself.. but did it the other way.. ... thought this would be more apt rather than giving the test id for the whole div.

Reply:
yeah it will cause conflict, but the new change more make sense i guess, we need to click `arrow icon` instead of the whole header